ebean-migration-db/src/main/java/io/ebean/migration/db/MigrationContextDb.java

```diff
@@ -0,0 +1,14 @@
+package io.ebean.migration.db;
+
+import io.ebean.Database;
+import io.ebean.Transaction;
+import io.ebean.migration.MigrationContext;
+
+/**
+ * @author Roland Praml, FOCONIS AG
+ */
+public interface MigrationContextDb extends MigrationContext {
+  public Transaction transaction();
+
+  public Database database();
```

Comment:
CHECKME: Should I add "Transaction" and "Database" in MigrationContext (means that ebean-migration will get a dependency to ebean-api)